runtime/acala/src/lib.rs

```diff
@@ -839,13 +839,15 @@ parameter_types! {
 	pub const GetStableCurrencyId: CurrencyId = AUSD;
 	pub const GetLiquidCurrencyId: CurrencyId = LDOT;
 	pub const GetStakingCurrencyId: CurrencyId = DOT;
+	pub AccountForTransfer: AccountId = AccountId::new([0u8; 32]);
```

Comment:
We should avoid using the 0 account id, it is not secure.
We can use `NetworkContractSource`

Reply:
`NetworkContractSource` in the runtime is `H160::from_low_u64_be(0)`, it'll be error when trying send to it. can we use `H160::from_low_u64_be(1)`?